aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -6,12 +6,13 @@ module aptos_framework::staking_config {
 
     friend aptos_framework::genesis;
 
-    /// Invalid required stake lockup value.
-    const EINVALID_LOCKUP_VALUE: u64 = 1;
-    /// Invalid rewards rate.
-    const EINVALID_REWARDS_RATE: u64 = 2;
-    /// Invalid required stake range, usually happens if min > max.
+    /// Stake lockup duration cannot be 0.
+    const EZERO_LOCKUP_DURATION: u64 = 1;
+    /// The denominator (representing reward rate fraction) cannot be 0.
+    const EZERO_REWARDS_RATE_DENOMINATOR: u64 = 2;
+    /// The specified stake range is invalid. Usually happens if min > max or max = 0.
```

Comment:
`Specified stake range is invalid.  Max must be greater than min.`

This technically covers both of your conditions, since max must be greater than min, and min's lowest value is 0.

Reply:
Updated